### ISSUE TITLE
Recovery phrase (mnemonic) length

### ISSUE THREAD

**Issue by sondreb** [block-core/blockcore#277]
The node UI will now generate a 24 word recovery phrase (mnemonic), perhaps this should be a user option and allow users to use 12 words if they want to?

**Comment by dangershony** [comment]
After thinking about this more I think its a good idea to allow 12 words.
While it perhaps has less entropy it's [still enough](https://bitcoin.stackexchange.com/questions/38512/is-12-word-seed-phrase-safe-enough#:~:text=In%20short%3A%2012%2Dword%20seed,some%20kind%20of%20control%20sum)

Mainly the benefit is the ability to remember the mnemonic which in some cases/users is more secure then storing it somewhere on disk or usb or safe.

So we should provide that option.

**Comment by soma42** [comment]
yes, so since blockcore serves a wide range of people, 12 and 24 should be supported...lets just hide the 'BIP-0039-Passphrase' under "advanced" or so.
